docs_source_files/config.toml

```diff
@@ -136,3 +142,17 @@ weight = 20
 name = "<i class='fas fa-fw fa-bullhorn'></i> How to contribute"
 url = "contributing"
 weight = 30
+[[Languages.fr.menu.shortcuts]]
```

Comment:
@Bredda could you please check the config file? Seems the `zh-cn` configuration and the `fr` one are overlapping. Please also check the output of `hugo server`, seems some links are broken.

Reply:
My bad. Fixed both conf file and links.
I should be ok now.

Reply:
Thanks!